docker-compose.jsonnet

```diff
@@ -21,7 +21,7 @@
     restreamer: true,
     backfiller: true,
     cutter: true,
-    sheetsync: false,
+    sheetsync: true,
```

Comment:
Should sheetsync be active by default?

Reply:
whoops, this was testing code that leaked in